src/vsr.zig

```diff
@@ -1337,27 +1338,29 @@ const ViewChangeHeadersArray = struct {
     }
 };
 
-/// For a replica with journal_slot_count=8 and lsm_batch_multiple=2:
+/// For a replica with journal_slot_count=9, lsm_batch_multiple=2, pipeline_prepare_queue_max=1, and
+/// checkpoint_interval = journal_slot_count - (lsm_batch_multiple + pipeline_prepare_queue_max) = 6
 ///
 ///   checkpoint() call           0   1   2   3
 ///   op_checkpoint               0   5  11  17
 ///   op_checkpoint_next          5  11  17  23
 ///   op_checkpoint_next_trigger  7  13  19  25
 ///
 ///     commit log (ops)           │ write-ahead log (slots)
-///     0   4   8   2   6   0   4  │ 0---4---
-///   0 ─────✓·%                   │ 01234✓6%   initial log fill
-///   1 ───────────✓·%             │ 890✓2%45   first wrap of log
```

Comment:
It was a bit confusing how, on the right, we show full wrap of the log (up to op=15), while on the left we are up to 13. In the new version, I made left and right to show the same data